docker/config_template.yaml

```diff
@@ -43,4 +44,9 @@ session:
   filesystem:
     # if non-empty, switches to filesystem store instead of cookie store. Increases size limit from 4K to 64K
     path: {{ default .Env.TEMPORAL_SESSION_STORE_PATH "" }}
-disableWriteActions: {{ default .Env.TEMPORAL_DISABLE_WRITE_ACTIONS "false" }}
+forwardHeaders:
+{{ if .Env.TEMPORAL_FORWARD_HEADERS }}
```

Comment:
What exactly is the expected env var here? An array of strings?

Reply:
added a comment